spanner/grpc_client.go

```diff
@@ -192,6 +192,15 @@ func (g *grpcSpannerClient) ExecuteStreamingSql(ctx context.Context, req *spanne
 	if mt != nil && client != nil && mt.currOp.currAttempt != nil {
 		md, _ := client.Header()
 		mt.currOp.currAttempt.setServerTimingMetrics(parseServerTimingHeader(md))
+		peerInfo, ok := peer.FromContext(client.Context())
```

Comment:
Can we move this code block into a small util method and call it from the three setreaming RPCs that need it, instead of repeating it for each RPC?